crates/rooch-framework/sources/address_type/ethereum_address.move

```diff
@@ -0,0 +1,74 @@
+module rooch_framework::ethereum_address {
+    use std::vector;
+    use std::error;
+    use rooch_framework::ecdsa_k1_recoverable;
+    use rooch_framework::hash;
+
+    // Ethereum addresses are always 20 bytes
+    const ETHEREUM_ADDR_LENGTH: u64 = 20;
```

Comment:
The const started with `E` will be treated as an error code. We need to add a prefix to it.

Reply:
Okay.

Reply:
I've fixed it.

Reply:
@jolestar 

Also is this needed to be fixed, since it's starting with `E`?

```move
/// there defines scheme for each blockchain
const ETHEREUM_SCHEME: u64 = 3;
```

Reply:
We discuss it in #663.

Reply:
> The const started with `E` will be treated as an error code. We need to add a prefix to it.

Reference:

Constant names: should be upper camel case and begin with an E if they represent error codes (e.g., EIndexOutOfBounds) and upper snake case if they represent a non-error value (e.g., MIN_STAKE).